fiftyone/operators/executor.py

```diff
@@ -157,11 +158,10 @@ def execute_operator(operator_uri, ctx=None, **kwargs):
         loop = None
 
     if loop is not None:
-        # @todo how can we await result here?
+        # @todo is it possible to await result here?
```

Comment:
Returning the coroutine is also an option, I think. Some discussion on [coroutines and tasks](https://stackoverflow.com/questions/57966935/asyncio-task-vs-coroutine)

```python
try:
    # Some contexts like notebooks already have a running event loop
    # return the coroutine for the user to await or schedule, if so
    loop = asyncio.get_running_loop()
    return coroutine
except:
    pass

result = asyncio.run(coroutine)
result.raise_exceptions()
return result
```